pkgs/servers/mautrix-whatsapp/default.nix

```diff
@@ -1,17 +1,19 @@
 { stdenv, buildGoModule, fetchFromGitHub }:
 
 buildGoModule {
-  pname = "mautrix-unstable";
-  version = "2019-09-03";
+  pname = "mautrix-whatsapp-unstable";
+  version = "2019-11-17";
```

Comment:
```
  pname = "mautrix-whatsapp";
  version = "unstable-2019-11-17";
```

Reply:
AFAICS `-unstable` should be a part of unstable, see https://github.com/NixOS/nixpkgs/issues/68531 and https://github.com/NixOS/nixpkgs/issues/68518.